### ISSUE TITLE
How do you make shortcut keys to be in toggle mode?

### ISSUE THREAD

**Issue by nukeCrayon** [opentoonz/opentoonz#4665]
### Ask a question related to OpenToonz

I am exploring opentoonz and just wondering how to make the spacebar shortcut key works like Krita or Harmony. 
Like when you hold the spacebar key, it will just be in pan mode while you still press it and then go back to your previous tool (let's say brush tool) when you are not holding the spacebar key.

Is it possible in opentoonz?

**Comment by shun-iwasawa** [comment]
Yes, exactly what you requested is already possible in OpenToonz. 
Holding the space key temporarily switches to the Hand Tool and allows you to pan the screen. Releasing the space key returns to the original tool.
Pressing and immediately releasing the space key switches the current tool (not temporarily) to the Hand Tool.

**Comment by nukeCrayon** [comment]
Hi Mr. Shun-iwasawa. I can't make it like that as of now in my system. I'm running Arch Linux. the toggle mode of spacebar is working in Krita but not in opentoonz.

**Comment by nukeCrayon** [comment]
here attached a video of what's happening.
[https://drive.google.com/file/d/17EjrJyGOgH_FRODRJRHGCeqmByBOMf-V/view?usp=sharing](url)

**Comment by gab3d** [comment]
The shortcut is working as intended in your system.

What you want is to be able to press and hold spacebar to temporarily activate the Pan Tool, without spacebar being the tool shortcut (i.e. that if you press and release spacebar nothing happen).
Am I right?

**Comment by nukeCrayon** [comment]
> What you want is to be able to press and hold spacebar to temporarily activate the Pan Tool, without spacebar being the tool shortcut (i.e. that if you press and release spacebar nothing happen). Am I right?

yes. that's what I want and trying to say. is there a settings on how to make it that way?

**Comment by andeon** [comment]
@RodneyBaker I tested it in the latest version of OpenToonz on Linux, and it works as expected:

- Pressing and holding the Space key temporarily switches to the Hand Tool for panning the view.
- Releasing the Space key returns to the previously selected tool (e.g., Brush Tool).

I think this issue can be closed. If the user still encounters the problem in version 1.7.1 or in the latest nightly build, they can reopen the issue or leave a comment here with more details.

**Comment by RodneyBaker** [comment]
Thanks @andeon
